src/utils/components/PendingChanges/hooks/usePendingChanges.tsx

```diff
@@ -80,6 +84,7 @@ export const usePendingChanges = (
     isList: true,
   });
 
+  const instanceTypeChanged = checkInstanceTypeChanged(vm, vmi);
```

Comment:
You introduce new IT change, what happen if u change IT? We are getting the message or IT change and memory and CPU? both of them?

Reply:
> what happen if u change IT?
>
notification is displayed - only for the instance type (as visible on the screenshot)